### PR TITLE
fix(icons): try to fix the unconstrained size in big sur #171

### PR DESCRIPTION
**Pull request by jgoldhammer**
### Status
**READY**

### Description
This PR should fix hopefully the issue 171. 

### Steps to Test or Reproduce

- Test with Big Sur
- Navigate to the preference dialog and open Apperance tab
- Click on the dropdown icon in the statusbar section

Due to usage of Catalina I cannot test it, but in other areas we also have set a size, so this will probably fix it.

### CODE REVIEW

File: MeetingBar/Views/Preferences/AppearanceTab.swift [hunk 1]
```diff
@@ -37,25 +37,25 @@ struct StatusBarSection: View {
...
+                        Image(nsImage: getImage(iconName: EventTitleIconFormat.appicon.rawValue)).resizable()
+                            .frame(width: 16.0, height: 16.0)
                         Text("\u{00A0}App icon")
                     }.tag(EventTitleIconFormat.appicon)
 
                     HStack {
-                        Image(nsImage: NSImage(named: EventTitleIconFormat.eventtype.rawValue)!).resizable()
-                                .frame(width: 16.0, height: 16.0)
+                        Image(nsImage: getImage(iconName: EventTitleIconFormat.eventtype.rawValue)).resizable()
+                            .frame(width: 16.0, height: 16.0)
```
Comment (leits) [inline MeetingBar/Views/Preferences/AppearanceTab.swift:53]: Do we need this `frame` if we modify NSImage size directly?

**Comment by leits** [comment]
Thank you for finding time to fix this, @jgoldhammer!

File: MeetingBar/Views/Preferences/AppearanceTab.swift (hunk 1 above)
Comment (jgoldhammer) [reply MeetingBar/Views/Preferences/AppearanceTab.swift:53]: I do not know really. Maybe it is the wrong way and we should do this?

`Use a Frame and aspectRatio set to .fit`
https://blckbirds.com/post/swiftui-how-to-frame-an-image/

Maybe we have to distribute test builds to let others test with Big Sur ??

Comment (leits) [reply MeetingBar/Views/Preferences/AppearanceTab.swift:53]: Ok. I'll find someone with Big Sure 👌

**Review by leits — APPROVED** [review]

**Pull request merged**